app/views/quotes/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <h2>
-  <span>SPLURTY / No. <%= @quote.id %></span>
+  <span>SPLURTY yoo boooyyys / No. <%= @quote.id %></span>

```

Comment:
Hey, I wouldn't make this change, let's try something else!

Reply:
Dang, you're right.